code/game/gamemodes/cult/cult_items.dm

```diff
@@ -682,3 +682,107 @@
 	desc = "It's an orb of crystalized blood. Can be used to transfer blood between cultists."
 	var/blood = 50
 
+/obj/item/portal_amulet
+	name = "reality sunderer"
+	icon = 'icons/obj/cult.dmi'
+	icon_state = "amulet"
+	desc = "Some amulet made out of metal, bluespace crystals, and blood. Allows cultists to open portals over teleport runes, destroying the rune in the process."
+	w_class = WEIGHT_CLASS_SMALL
+
+
+/obj/item/portal_amulet/afterattack(atom/O, mob/user, proximity)
+	. = ..()
+	if(!iscultist(user) && ishuman(user))
```

Comment:
Is there a case where a non-human uses this item? If so then they will be able to use the item without issues. Dunno if that is currently possible though

Reply:
I believe the ishuman was needed for the eyeflash to work properly, but I will double check